controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/ContainerControllerTester.java

```diff
@@ -88,6 +88,12 @@ public Application deploy(Application application, ApplicationPackage applicatio
     }
 
     public void notifyJobCompletion(ApplicationId applicationId, long projectId, boolean success, DeploymentJobs.JobType job) {
+        try {
+            Thread.sleep(1);
+        }
+        catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
This as well.

Reply:
Well, not the Exception, but the sleep(1) above it.

Reply:
To make sure a measurable amount of time (1ms) has passed between triggering and completion of a job — same as for the DeploymentTester.notifyJobCompletion

Reply:
Tests using this tester should not care about actual time.

Reply:
Yes, of course they should; the clock in the controller is a real one in this case.